### ISSUE TITLE
[F 6.0.1] Link in submenu not working.

### ISSUE THREAD

**Issue by Gacnt** [zurb/foundation-sites#7277]
I have this bit of code:

```
<div class="top-bar-right">
			<ul class="dropdown menu" data-dropdown-menu>
				<li>
					<a href="#">{{username}}</a>
					<ul class="menu">
						<li>
							<a href="/auth/logout">Logout</a>
						</li>
					</ul>
				</li>
			</ul>
		</div>
```

When I click Logout, nothing happens the menu closes but it just sits still.. If I manually copy the `href` into my address bar it works just fine. JS console is also saying: `Component not defined!` (only with the `.min` version, I don't get any warning with the un-minified version)

If I include `foundation.dropdownMenu.js` it makes my dropdown menu open in a weird spot like way off the screen, but the link does work. I don't know what to do to fix this. Breaking my site thanks.

**Comment by Gacnt** [comment]
Bump

**Comment by zurbchris** [comment]
v6.0.1 min files still had a mangling issue that was preventing the components from loading correctly. Also, there was an issue with menu items not following links on clicks in dropdown menus. This should be corrected in the current version. Drilldowns are still a bit... off, we'll say. 
There is a question I've asked about what a dropdown should do on multiple clicks/taps here: 
https://github.com/zurb/foundation-sites/issues/7301 
There will likely be a new option added to determine what secondary clicks do for submenu parent li's, but, in the near future. 
Please note: you have to include the proper js for the plugins to work, so if you're using a custom download or package manager install, make sure you're using the full js file or selective importing them in the proper order.

**Comment by zurbchris** [comment]
I'm going to mark this closed since 6.0.4 solved issue you listed.
